server/src/internalClusterTest/java/org/opensearch/indices/stats/IndexStatsIT.java

```diff
@@ -143,6 +144,12 @@ public Settings indexSettings() {
             .build();
     }
 
+    @Override
+    // Do not parameterize this test, instead let individual test methods modify concurrent search settings as needed.
```

Comment:
Why is that limitation?

Reply:
While the tests in this class do execute search requests, the metrics aren't gathered from within the queryphase itself but rather before and after it, so I don't really see a point in executing it with concurrent search on and off.

With the introduction of concurrent search related metrics, I think it would be better to have tests like the one I added that execute requests with and without concurrent search in the same test itself and check if the results are accumulating correctly when doing so. Unfortunately, this behavior is not currently supported by the parameterized test class either so I think it makes more sense to do it this way.

I can update the code comment to be more clear though.

Reply:
May be we could just spin it off into another test case (feature flags + testConcurrentQueryCount) and keep this one unchanged?

Reply:
Sure that works for me, I will make that change